src/commands/kv/bucket/upload.rs

```diff
@@ -22,9 +22,25 @@ pub fn upload(
     user: GlobalUser,
     namespace_id: &str,
     path: &Path,
+    verbose: bool,
+) -> Result<(), failure::Error> {
+    match upload_files(target, user, namespace_id, path, verbose) {
```

Comment:
I have `upload()` wrap `upload_files()` to ensure that callers of `upload_files()` (like `sync()`) don't have Success messages printed out for them (this would lead to duplicative Success messages). `upload()` handles this printing instead.